File: python/liczby23.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  liczby23.py

def liczby2():
    """
    Funkcja drukuje wszystkie liczby dwucyfrowe, w których nie powtarzają 
    się cyfry.  NA KONIEC ZWRACA ILOŚĆ takich liczb wykluczone liczby: 11 22 
    """
    ile = 0 #licznik liczb 
    for i in range(1, 10): #pętla dziesiątek 
        for j in range(10): #pętla jednostek 
            if i != j:
                print("{}{} ".format(i, j), end="")
                ile = ile + 1 #zliczanie liczb 
    return ile


def main(args):
    print("Liczba 2-cyfrowych:", liczby2())
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

def liczby3():
    
    ile = 0 #licznik liczb 
    for i in range(1, 10): #pętla setek 
        for j in range(10): #pętla dziesiątek
            for k in range(10): #pętla jednostek 
            if i != j: and != k and j != k:
                print("{}{}{} ".format(i, j, k), end="")
                ile = ile + 1 #zliczanie liczb 
    return ile 
    
def main(args):
    print ("\n\nLiczb 2 cyfrowych:", liczby 2 

File: cpp/liczby23.cpp
/*
 * liczby23.cpp
 */


#include <iostream>
using namespace std;

int liczby2() {
    int ile = 0;
    for(int i = 1; i < 10; i++) {
        for(int j = 1; j < 10; j++) {
            if(i != j) {
               cout << i << j << " ";
               ile ++;
            } 
        }
    }
    return ile;
}


int main(int argc, char **argv)
{
	cout << "\n\nLiczb 2 cyfrowych:" << liczby2() ;
	return 0;
}

